setup.py

```diff
@@ -39,6 +39,7 @@
 ]
 
 grpc_deps = ['grpcio~=1.35.0',
+             'grpcio-tools~=1.35.0',
```

Comment:
why do we need this dependency?

Reply:
I had to install it, but I can't remember which part required it now!